Earthfile

```diff
@@ -366,6 +373,7 @@ image:
         ARG OS_VERSION=${KAIROS_VERSION}
     END
     ARG OS_ID
+    # should we add the model to the resulting iso?
```

Comment:
you mean in `/etc/os-release`? Consider moving it here: https://github.com/kairos-io/kairos/issues/1365